src/Compilers/CSharp/Portable/Compilation/MemberSemanticModel.cs

```diff
@@ -1087,6 +1087,17 @@ public override TypeInfo GetTypeInfo(SelectOrGroupClauseSyntax node, Cancellatio
         private void GetBoundNodes(CSharpSyntaxNode node, out CSharpSyntaxNode bindableNode, out BoundNode lowestBoundNode, out BoundNode highestBoundNode, out BoundNode boundParent)
         {
             bindableNode = this.GetBindableSyntaxNode(node);
+
```

Comment:
> it looks like GetBindableSyntaxNode can return node that is outside of Root node for this member semantic model

that seems bad.  can you hit that for any other SemanticModel operations?  If not, how do they avoid this problem?